src/components/Microsoft.DotNet.UpgradeAssistant/Checks/WcfServerCheck.cs

```diff
@@ -16,9 +16,10 @@ namespace Microsoft.DotNet.UpgradeAssistant.Checks
     /// </summary>
     public class WcfServerCheck : IUpgradeReadyCheck
     {
-        private const string UPGRADE_LINK = "https://aka.ms/migrate-wcf-to-grpc";
+        private const string WCF_LINK = "https://aka.ms/CoreWCF/migration";
+        private const string gRPC_LINK = "https://aka.ms/migrate-wcf-to-grpc";
```

Comment:
Build's failing cuz of this
```suggestion
        private const string GRPC_LINK = "https://aka.ms/migrate-wcf-to-grpc";
```